src/pages/home/report/EmojiPickerMenuItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Pressable, Text} from 'react-native';
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    // The unicode that is used to display the emoji
+    emoji: PropTypes.string.isRequired,
+
+    // The function to call when an emoji is selected
+    onPress: PropTypes.func.isRequired,
+};
+
+const EmojiPickerMenuItem = props => (
+    <Pressable
+        onPress={() => props.onPress(props.emoji)}
+        style={({hovered}) => ([
```

Comment:
Can you also give this a `pressed` style like so:

```javascript
import styles, {getButtonBackgroundColorStyle} from '../../../styles/styles';
import getButtonState from '../libs/getButtonState';

style={({hovered, pressed) => ([
    styles.emojiItem,
    getButtonBackgroundColorStyle(getButtonState(hovered, pressed)),
])}
```

I've been trying to make this behavior more consistent across the app (see [this PR](https://github.com/Expensify/Expensify.cash/pull/2232)...)